src/main/java/io/candydoc/plugin/save_ubiquitous_language/file/UbiquitousLanguageAsFile.java

```diff
@@ -57,4 +58,33 @@ public static UbiquitousLanguageAsFile forFormat(String outputFormat, String out
     }
     throw new IllegalArgumentException("Supported file formats are 'json', 'yml'.");
   }
+
+  private Path writeToFile(UbiquitousLanguage ubiquitousLanguage) throws IOException {
+    UbiquitousLanguageDto dto = UbiquitousLanguageDto.from(ubiquitousLanguage);
+    String fileName = getFileName(dto.getName());
+    serializer.writeValue(destinationFolder.resolve(fileName).toFile(), dto);
+    return Path.of("./", fileName);
+  }
+
+  private String getFileName(String name) {
+    return name.toLowerCase().replace(" ", "_") + "." + extension;
+  }
+
+  private class Manifest {
+    private final ManifestDto manifestDto;
```

Comment:
A Manifest depending on a ManifestDTO ? IMO, it does not make any sens. I think the Manifest can do the job on its own.